taxi/forms.py

```diff
@@ -10,6 +10,8 @@ class CarForm(forms.ModelForm):
     drivers = forms.ModelMultipleChoiceField(
         queryset=get_user_model().objects.all(),
         widget=forms.CheckboxSelectMultiple,
+        required=False,
+        blank=True,
```

Comment:
The `blank=True` attribute is not valid for form fields. It is used in model fields to specify whether the field is allowed to be empty in the database. In forms, you should use `required=False` to indicate that the field is optional. You can remove `blank=True` from this line.